gramps_webapi/api/emails.py

```diff
@@ -52,15 +52,15 @@ def email_confirm_email(base_url: str, token: str):
     """Confirm e-mail address e-mail text."""
     intro = (
         _(
-            "You are receiving this email because this e-mail address "
+            "You are receiving this message because this e-mail address "
```

Comment:
I just meant the inconsistent spelling (email vs e-mail), but good like this :blush:

Reply:
Doh! Didn't pick up on that :)